### ISSUE TITLE
Convert to big.Int

### ISSUE THREAD

**Issue by sammy007** [shopspring/decimal#165]
There is `IntPart`, could you add the same but returning `big.Int`?

**Comment by mwoss** [comment]
Hi @sammy007. Thanks for the suggestion, but do you think such functionality will be useful for the community? Implementing additional method for a single person is not a good idea.

**Comment by treeder** [comment]
I think this would be nice to have as well. Otherwise you have to do the following:

```go
d := decimal.New(1,0)
i := &big.Int{}
i.SetString(d.StringFixed(0), 10)
return i
```

A `BigIntPart()` function might be nice to wrap that up and maybe more importantly, prevent people from making mistakes like:

```go
return big.NewInt(d.IntPart()) // BAD
```

**Comment by mwoss** [comment]
Two requests, it seems reasonable to implement then :D
We will add this request to our backlog and we try to implement it as soon as possible.

**Comment by treeder** [comment]
While we're at it, a `BigFloat()` function would be nice too.

**Comment by mwoss** [comment]
Implemented in #171.

**Comment by mwoss** [comment]
Closing, new functionalities released along with v1.2.0 :)
